### DISCUSSION TITLE
APQ Support

### DISCUSSION THREAD

**Discussion by vineela-gampa** [graphql-java/graphql-java#2567] [Q&A]
We would like to use APQ for grapqhl. Can you please suggest the version APQ is implemented into.

**Comment by bbakerman** [comment]
What is APQ?

I quick google revealed nothing

**Comment by vineela-gampa** [comment]
i was referring to Automated persisted queries . https://www.apollographql.com/docs/apollo-server/performance/apq/

**Comment by bbakerman** [comment]
The class `graphql.execution.preparsed.persisted.ApolloPersistedQuerySupport` has support for reading the persisted hash from "query extensions"

It does not use a version (since there is no other version published)
